crates/gitbutler-core/src/remotes/controller.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{
+    error::Error,
+    project_repository,
+    projects::{self, ProjectId},
+};
+
+#[derive(Clone)]
+pub struct Controller {
+    projects: projects::Controller,
+}
+
+impl Controller {
+    pub fn new(projects: projects::Controller) -> Self {
+        Self { projects }
+    }
+
+    pub async fn remotes(&self, project_id: &ProjectId) -> Result<Vec<String>, Error> {
```

Comment:
`&ProjectId` should be `&ProjectId` as it's `Copy`.

Reply:
Hey! Sorry, it looks like you've said `&ProjectId` twice. Could you clarify what you mean perhaps by adding a suggestion block?

Reply:
Apologies, I meant Project, without ampersand.